src/plane.rs

```diff
@@ -220,12 +220,38 @@ impl Plane {
   }
 }
 
+#[derive(Clone, Copy)]
 pub struct PlaneSlice<'a> {
   pub plane: &'a Plane,
   pub x: isize,
   pub y: isize
 }
 
+pub struct IterWidth<'a> {
+    ps: PlaneSlice<'a>,
+    width: usize,
+}
+
+// TODO: Implement more methods
```

Comment:
Could you just clarify what methods you are referring to, here?

Reply:
OK, I was just wondering if it was worth to indicate the methods that would be useful to implement in this particular case within the comment. If we want to implement all remaining methods then I guess it is not so important.

Reply:
If that means we no longer have a clear intent to add more methods then we should drop the TODO.

Reply:
Done :)